processes/executive-copy-creation-instructions.md

```diff
@@ -0,0 +1,128 @@
+| _Example_  | _**Authorization**:_ `[Support Scope 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)`
+
+|            |`$link_to_proposal`|
+| ---------- | :---------------- |
+| Guidelines | Provide the link to the last version of the **Proposal** presenting the item and providing reasoning for including it in the Executive Vote (see the _Reasoning URL_ in the [EC sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?pli=1#gid=1593813984)).|
+| _Example_  | _**Proposal**_: `[Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-4-q2-2024/24572)` 
+
+|            |`$executive_entry_description_`|
+| ---------- | :---------------------------- |
+| Guidelines | Provide a high-level overview of **all actions** contained in the Proposal, framing them in future-conditional tense (_IF this executive proposal passes, THEN this action will happen_).<br></br>Include **required hyperlinks**, where applicable and available:<br><li>Links to Ethereum and IPFS addresses.</li><li>Links to any analysis that has been published by DAO-funded entities.</li><li>Links to well-maintained resources describing the nature of any parameters that may be changed if the executive proposal passes.</li><li>Any other links providing additional checks, references, or further context surrounding the inclusion of the item in the executive proposal.<ul>|
+| _Example_  | _If this executive proposal passes, a payment of **219,125 DAI** will be made to a smart contract controlled by Aave Governance at `[0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c)` as part of the Aave-SparkLend Revenue Share._
+
+
+8. Review the completed draft against
+    - The [Executive Copy Drafting Principles]($).
+    - The [Executive Copy Creation Checklist](https://github.com/makerdao/community/blob/master/processes/executive-copy-creation-checklist.md).
+
+>[!IMPORTANT]
+> Make sure all items in the [Executive Contents sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?pli=1#gid=1593813984) have been confirmed by the content liaisons.
+
+## Submit Pull Request
+
+9. Commit changes, and create a Pull Request (PR) to the Community GitHub repository.
+
+10. Inform the Reviewer(s) that the Executive Vote copy is ready for their review. 
+
+11. Ensure the approved Executive Vote copy PR is merged.
+
+12. Inform technical teams that the Executive Vote copy is ready and provide the link.
+
+13. Once technical teams have provided a spell address, add it to the Executive Vote copy.
+
+## Post Executive Vote
```

Comment:
Should this section live in this document? Seems like the rest of this text is focused on copy creation. IMO, these steps should be on the executive submission checklist, as it will be the submitter's job to handle the address and portal items.